frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/create-sc/wizard-pages/create-local-volume-set.tsx

```diff
@@ -103,6 +104,20 @@ const ConfirmationModal = ({ state, dispatch, setInProgress, setErrorMessage, ns
     dispatch({ type: 'setShowConfirmModal', value: false });
   };
 
+  const description = (
+    <>
+      <span>
+        {
+          "After the volume set and storage class are created you won't be able to go back to this step."
+        }
```

Comment:
```suggestion
After the volume set and storage class are created you won't be able to go back to this step.
```

Reply:
Required for the translations! Afreen's suggestion otherwise it will give linter error on `&apos;`